This patch originated in-house but is not suitable for upstream.

--- meld-1.8.6/INSTALL.orig	2015-03-13 11:10:19.822910199 -0700
+++ meld-1.8.6/INSTALL	2015-03-13 11:10:40.322859285 -0700
@@ -33,8 +33,8 @@
 ## 
 ## If you need to, you can customise the install paths used here:
 ##
-PYTHON ?= python
-prefix := /usr/local
+PYTHON ?= python2.7
+prefix := /usr
 bindir := $(prefix)/bin
 libdir := $(prefix)/lib
 docdir := $(prefix)/share/doc
